bigquery/user_defined_functions/statistical-functions.sql

```diff
@@ -0,0 +1,43 @@
+  # Contains useful statistical functions that can be reused across multiple scheduled queries and views in BigQuery.
+  # This script must be run in BigQuery for these functions to become available, and re-run to overwrite the available functions with any new version.
+  #
+  # Test whether there is a probability greater than a specified confidence_level that two probabilities of a Boolean-valued outcome calculated from two independent samples are different
+  # e.g. test whether there is a >95% probability that the probabilities of a vacancy having a particular tag on a particular date and the same date in the previous year are different
+  # x1,x2 are the numbers of true outcomes (e.g. heads, vacancies with tag) in each sample of size n1,n2
+CREATE OR REPLACE FUNCTION
+  `teacher-vacancy-service.production_dataset.two_proportion_z_test`(confidence_level STRING, x1 INT64, n1 INT64, x2 INT64, n2 INT64)
```

Comment:
Is it possible to have more transparent parameter names than 'x1' and 'n2'?

Reply:
@david-mears-dfe yes, I'll rename them as the numbers of successes and trials in each sample.